python/hail/expr/functions.py

```diff
@@ -537,6 +537,23 @@ def exp(x) -> Float64Expression:
     return _func("exp", tfloat64, x)
 
 
+@typecheck(ac=expr_int32, an=expr_int32, ci=expr_float64)
```

Comment:
Also in experimental.

Reply:
Whoops, meant to move this and forgot. Will do.